### PR TITLE
Network switch and multiple wallet support added

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
- Added multiple wallet and network switch support

### Note : 
1. Private net is only for testing purpose, also wallet connect and myalgo wallet doesn't support it 
2. Changed `webMode` type to `any` in `fundAmount` since all wallets mode have different type after connection which aren't specified in web package i.e it is different from `WebModeTypes`
3. I have not updated the indexer network since we will upgrade to local indexer.

image

### CODE REVIEW

File: src/utility/algod.utility.ts [hunk 1]
```diff
@@ -11,9 +11,9 @@ export const fundAmount = async (
 	from: string,
 	to: string,
 	amount: number,
-	webMode: WebMode
+	webMode: any // eslint-disable-line
```
Comment (rmeena840) [inline src/utility/algod.utility.ts:14]: Can't we make it type `WebModeTypes`?

**Review by rmeena840 — APPROVED** [review]
Good work!! LGTM. Left one comment. Pre-approving.

**Comment by rmeena840** [comment]
@Megha-Dev-19 You can now delete this file `algosigner.config`. We are getting NetworkType from enum now.

File: src/utility/algod.utility.ts (hunk 1 above)
Comment (Megha-Dev-19) [reply src/utility/algod.utility.ts:14]: No we can’t because after connecting the wallet new type is assigned to my algo connect and wallet connect

**Comment by Megha-Dev-19** [comment]
> @Megha-Dev-19 You can now delete this file `algosigner.config`. We are getting NetworkType from enum now.

Yeah sure, your local indexer PR wasn’t merged  so I didn’t do it

**Pull request merged**